src/comparisons.jl

```diff
@@ -1,7 +1,42 @@
-# This file is inspired from Polyhedra.jl
+# Some functions in this file are inspired from Polyhedra.jl
 
-const ABSZTOL(N::Type{<:AbstractFloat}) = N(10) * sqrt(eps(N))
-const ABSZTOL(N::Type{Rational{INNER}}) where {INNER} = zero(N)
+# struct to contain the tolerances for a given numberic type
+mutable struct TOL{N<:Number}
```

Comment:
Let us not use capital letters for types.
```suggestion
mutable struct Tolerance{N<:Number}
```

Reply:
Good, i changed it.